packages/network-of-terms-catalog/catalog/queries/search/gtaa-persoonsnamen.rq

```diff
@@ -62,20 +35,6 @@ WHERE {
         ?uri skos:scopeNote ?scopeNote .
         FILTER(LANG(?scopeNote) = "nl")
     }
-    OPTIONAL {
-        ?uri skos:broader ?broader_uri .
-        ?broader_uri skos:prefLabel ?broader_prefLabel .
-        FILTER(LANG(?broader_prefLabel) = "nl")
-    }
-    OPTIONAL {
-        ?uri skos:narrower ?narrower_uri .
-        ?narrower_uri skos:prefLabel ?narrower_prefLabel .
-        FILTER(LANG(?narrower_prefLabel) = "nl")
-    }
-    OPTIONAL {
-        ?uri skos:related ?related_uri .
-        ?related_uri skos:prefLabel ?related_prefLabel .
-        FILTER(LANG(?related_prefLabel) = "nl")
-    }
```

Comment:
Did you remove broader, narrower and related because Persoonsnamen doesn’t have those relations or only to limit the number of triples returned per name in order to speed up the query?